bioconda_utils/cli.py

```diff
@@ -309,10 +305,16 @@ def lint(recipe_folder, config, packages="*", cache=None, list_funcs=False,
 @arg('--keep-image', action='store_true', help='''After building recipes, the
      created Docker image is removed by default to save disk space. Use this
      argument to disable this behavior.''')
-@arg('--prelint', action='store_true', help='''Just before each recipe, apply
+@arg('--lint', '--prelint', action='store_true', help='''Just before each recipe, apply
```

Comment:
Left `--prelint` as an alias here so that this can be swapped out without issues. The alias should be removed when `bioconda-recipes` doesn't use it anymore.